custom_components/myskoda/device_tracker.py

```diff
@@ -84,8 +84,13 @@ def longitude(self) -> float | None:  # noqa: D102
     def extra_state_attributes(self) -> dict:
         """Return extra state attributes."""
         attributes = {}
-        if render := self.get_renders().get("main"):
-            attributes["entity_picture"] = render
+
```

Comment:
would you keep `renders` as main source of the image and use `composite_renders` as backup?

Reply:
yep this sounds like a good idea to keep main and use composite as fallback, I will change it

Reply:
The composite_renders are useful as they represent the stacks of images in various situations. If you look at the data, you see there are various situations in the yaml: home, charging_light, charging_dark, etc. The list of exterior_side entries in that section represent the various (partial) images in the order they are stacked.

Reply:
@gschot I know, but expecting that unmodified one will be the one where is nothing changed or stacked... this is just about changing entity picture for main render and device tracker as MySkoda api is sending now empty list for renders.. To have entity picture present in such a case, we select another picture from composite renders list.. (and my best guess is that unmodified will be always present and always single)

Reply:
> yep this sounds like a good idea to keep main and use composite as fallback, I will change it

If you need to do a fallback, please log a debug message

Reply:
@prvakt Got it! Thanks for the clarification.